### PR TITLE
Fixes #RHINENG-1793 - transform provider name 'google' to 'gcp'

### PR DESCRIPTION
**Pull request by kgaikwad**

### CODE REVIEW

**Comment by ShimShtein** [comment]
LGTM code-wise.

**Comment by kgaikwad** [comment]
\retest

**Review by patilsuraj767 — APPROVED** [review]
I don't have any archive with gcp instance to test changes locally, but looking at the code LGTM 👍

File: yuptoo/modifiers/transform_cloud_provider.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+class TransformCloudProvider(Modifier):
+    def run(self, host: dict, transformed_obj: dict, **kwargs):
+        """Transform 'system_profile.cloud_provider.
+        When system_profile.cloud_provider is 'google',
+        transform it to 'gcp' for consistency.
+        """
+        system_profile = host.get('system_profile', {})
+        cloud_provider = system_profile.get('cloud_provider')
+        if cloud_provider is not None and 'google' in cloud_provider.lower():
```
Comment (r14chandra) [inline yuptoo/modifiers/transform_cloud_provider.py:16]: Are we just processing archives from `GCP`?
I think we need to include others as well.

**Review by r14chandra — CHANGES_REQUESTED** [review]

File: yuptoo/modifiers/transform_cloud_provider.py (hunk 1 above)
Comment (kgaikwad) [reply yuptoo/modifiers/transform_cloud_provider.py:16]: As per offline [conversation](https://redhat-internal.slack.com/archives/C018YNTB203/p1693997757723339), the issue is only systems from GCP.

**Comment by kgaikwad** [comment]
Thank you @ShimShtein @patilsuraj767  and @r14chandra  for review and comments.

**Pull request merged**